sections/Header/Header.tsx

```diff
@@ -1,69 +1,77 @@
-//@ts-nocheck
 import { FC, useMemo } from 'react';
 import Image from 'next/image';
 import styled from 'styled-components';
 import Link from 'next/link';
 
 import AelinLogo from 'assets/svg/aelin-logo.svg';
+import UniswapLogo from 'assets/svg/uniswap-logo.svg';
+import GitbookLogo from 'assets/svg/gitbook-logo.svg';
 import ROUTES from 'constants/routes';
 import WalletWidget from 'components/WalletWidget';
 import NetworkWidget from 'components/NetworkWidget';
-import { FlexDiv, ExternalLink } from 'components/common';
+import { ExternalLink, FlexDivCentered } from 'components/common';
 
 const Header: FC = () => {
 	const LINKS = useMemo(
 		() => [
 			{ label: 'Pools', pathname: ROUTES.Pools.Home },
-			{ label: 'Claim Aelin', pathname: ROUTES.ClaimTokens },
 			{ label: 'Stake', pathname: ROUTES.Stake },
-			{ label: 'Uniswap Pool', pathname: ROUTES.UniswapPool, newTab: true },
-			{ label: 'Docs', pathname: ROUTES.Docs, newTab: true },
+			{ label: 'Claim', pathname: ROUTES.ClaimTokens },
+			{ label: 'Buy Aelin', pathname: ROUTES.UniswapPool, newTab: true, image: UniswapLogo },
+			{ label: 'Docs', pathname: ROUTES.Docs, newTab: true, image: GitbookLogo },
 		],
 		[]
 	);
 	return (
 		<Container>
-			<ImageContainer>
-				<Link href={ROUTES.Home}>
-					<a>
-						<StyledImage src={AelinLogo} alt="aelin logo" width={98} height={22} />
-						<BetaLabel>[alpha]</BetaLabel>
-					</a>
-				</Link>
-			</ImageContainer>
+			<Content>
+				<FlexDivCentered>
+					<Link href={ROUTES.Home}>
+						<a>
+							<StyledImage src={AelinLogo} alt="aelin logo" width={98} height={22} />
+						</a>
+					</Link>
 
-			<Links>
-				{LINKS.map(({ label, pathname, query, isDisabled, newTab }) => {
-					return newTab ? (
-						<StyledExternalLink key={`link-${label}`} href={pathname}>
-							{label}
-						</StyledExternalLink>
-					) : (
-						<Link
-							href={query != null ? { pathname, query } : { pathname }}
-							key={`link-${label}`}
-							passHref
-						>
-							<StyledLink className={isDisabled ? 'is-disabled' : ''} target="_self">
-								{label}
-							</StyledLink>
-						</Link>
-					);
-				})}
-			</Links>
-			<HeaderBlock>
-				<NetworkWidget />
-				<WalletWidget />
-			</HeaderBlock>
+					<Links>
+						{LINKS.map(({ label, pathname, newTab, image }) => {
+							return newTab ? (
+								<StyledExternalLink key={`link-${label}`} href={pathname}>
+									<>
+										<ExternalLinkLabel>{label}</ExternalLinkLabel>
+										<Image src={image} height={24} width={24} alt={`${label} logo`} />
```

Comment:
Is the image field always required when it comes to an external link?

I think a validation of the image field should be enough. Something like: 
`{image && <Image src={image} height={24} width={24} alt={`${label} logo`} />} `

Reply:
yes for now it is